README.md

```diff
@@ -21,49 +21,77 @@ FaceChain is powered by [ModelScope](https://github.com/modelscope/modelscope).
 
 # Installation
 
-You may use pip and conda to build a local python environment. We recommend using [Anaconda](https://docs.anaconda.com/anaconda/install/) to manage your dependencies. After installation, execute the following commands:
+## Compatibility Verification
+The following are the environment dependencies that have been verified:
+- python: py3.8, py3.10
+- pytorch: torch2.0.0, torch2.0.1
+- tensorflow: 2.7.0, tensorflow-cpu
+- CUDA: 11.7
+- CUDNN: 8+
+- OS: Ubuntu 20.04, CentOS 7.9
+- GPU: Nvidia-A10 24G
 
-```shell
-conda create -n facechain python=3.8    # python version >= 3.8
-conda activate facechain
+## Resource Usage
+- GPU: About 19G
+- Disk: About 50GB
+
+## Installation Guide
+The following installation methods are supported:
 
-pip3 install -r requirements.txt
-pip3 install -U openmim 
-mim install mmcv-full==1.7.0
-````
+1. Docker【recommended】
 
-You may use the official docker-image provided by ModelScope:
+If you are familiar with using docker, we recommend to use this way:
 
 ```shell
-registry.cn-hangzhou.aliyuncs.com/modelscope-repo/modelscope:ubuntu20.04-cuda11.7.1-py38-torch2.0.1-tf1.15.5-1.8.0
-```
-With this docker image, the only thing you need to install is Gradio.
+# Step1
+Prepare the env with GPU, e.g. Alibaba Cloud ECS, refer to: https://www.aliyun.com/product/ecs
```

Comment:
In addition to ModelScope notebook and ECS, I would suggest that we add that user may also start DSW instance with the option of ModelScope (GPU) image, to create a ready-to-use envrionment.